### PR TITLE
Improved routing section

### PR DESCRIPTION
**Pull request by HeahDude**
Continuation of #11113 in 4.2.

### CODE REVIEW

File: routing.rst [hunk 1]
```diff
@@ -27,7 +27,20 @@ route that can match *any* URL like ``/blog/my-post`` or
 ``/blog/all-about-symfony``.
 
 Routes can be configured in YAML, XML, PHP or annotations. All formats provide
-the same features and performance, so choose the one you prefer:
+the same features and performance, so choose the one you prefer. If you choose
+PHP annotations, run this command once in your app to add support for them:
```
Comment (javiereguiluz) [inline routing.rst]: We recently started replacing all "app" by "application".

File: routing.rst [hunk 2]
```diff
@@ -27,7 +27,20 @@ route that can match *any* URL like ``/blog/my-post`` or
...
+
+.. code-block:: terminal
+
+    $ composer require annotations
+
+.. note::
```
Comment (javiereguiluz) [inline routing.rst]: I wouldn't add such a detailed note ... maybe we can move this to a phrase wrapped in parenthesis after the _"... run this command once in your app to add support for them:"_

File: routing/external_resources.rst [hunk 3]
```diff
@@ -88,8 +88,8 @@ Prefixing the URLs of Imported Routes
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 You can also choose to provide a "prefix" for the imported routes. For example,
-suppose you want to prefix all application routes with ``/site`` (e.g.
-``/site/blog/{slug}`` instead of ``/blog/{slug}``):
+to prefix all application routes with ``/site`` (e.g.``/site/blog/{slug}``
```
Comment (OskarStark) [inline routing/external_resources.rst]:
```suggestion
to prefix all application routes with ``/site`` (e.g. ``/site/blog/{slug}``
```

**Review by OskarStark — CHANGES_REQUESTED** [review]

File: routing.rst (hunk 1 above)
Comment (HeahDude) [reply routing.rst]: Fixed.

File: routing.rst (hunk 2 above)
Comment (HeahDude) [reply routing.rst]: Done.

File: routing/external_resources.rst (hunk 3 above)
Comment (HeahDude) [reply routing/external_resources.rst]: Fixed.

File: security/form_login_setup.rst [hunk 4]
```diff
@@ -74,6 +74,51 @@ class that processes the login submit and 4) updates the main security config fi
         }
     }
 
+Edit the ``security.yml`` file in order to allow access for anyone to the
```
Comment (OskarStark) [inline security/form_login_setup.rst]: security.yaml

File: security/form_login_setup.rst [hunk 5]
```diff
@@ -74,6 +74,51 @@ class that processes the login submit and 4) updates the main security config fi
...
+
+    .. code-block:: xml
+
+        <!-- config/packages/security.xml -->
+        <?xml version="1.0" charset="UTF-8" ?>
+        <srv:container xmlns="http://symfony.com/schema/dic/security"
+            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+            xmlns:srv="http://symfony.com/schema/dic/services"
+            xsi:schemaLocation="http://symfony.com/schema/dic/services
+                http://symfony.com/schema/dic/services/services-1.0.xsd">
```
Comment (OskarStark) [inline security/form_login_setup.rst]: HttpS for xsd url

**Review by OskarStark — COMMENTED** [review]
Should be fixed in 3.4 too

**Comment by HeahDude** [comment]
@OskarStark If this needs to be fixed in 3.4 I'll do it in another PR.

**Review by OskarStark — APPROVED** [review]

**Review by maxhelias — APPROVED** [review]

**Review by OskarStark — APPROVED** [review]

**Comment by javiereguiluz** [comment]
Jules, I'm sorry it took us so long to merge this. Thanks for your contribution!

**Pull request merged**